src/Native/Unix/System.Security.Cryptography.Native/pal_evp_cipher.c

```diff
@@ -131,6 +200,16 @@ const EVP_CIPHER* CryptoNative_EvpAes128Cbc()
     return EVP_aes_128_cbc();
 }
 
+const EVP_CIPHER* CryptoNative_EvpAes128Gcm()
+{
+    return EVP_aes_128_gcm();
```

Comment:
If you change the portable loader to say this is optional then this needs to be  `API_EXISTS(EVP_aes_128_gcm) ? EVP_aes_128_gcm() : NULL;`.  At which point the separate "Exists" method is redundant with calling this and checking for IntPtr.Zero.